CONTRIBUTING.md

```diff
@@ -23,6 +23,10 @@ Please take a look at this list first, your contributions may belong in one of t
    programmatic instrumentations that are now in the `ext` directory in the main OpenTelemetry repo. Please ask in the Gitter
    channel (see below) for guidance if you want to contribute with these instrumentations.
```

Comment:
Looks like still there is some old `ext` reference in this doc and they are no longer part of this repo, can we get that corrected in this PR itself?

Reply:
This PR is for introducing the metrics branch, not for any bug fixes. I will update the description.